tests/gen_tests.in

```diff
@@ -339,6 +339,7 @@ ioctl_kd-success-s1024-Xverbose	+ioctl-success.sh -a20 -s1024 -Xverbose
 ioctl_kvm_run	+ioctl.test -a36 -y
 ioctl_kvm_run-v	+ioctl.test -v -a36 -y
 ioctl_kvm_run_auxstr_vcpu	+ioctl.test -a36 -y -e kvm=vcpu
+ioctl_lirc-success	+ioctl-success.sh -a30
```

Comment:
Please add
```
ioctl_lirc	+ioctl-success.sh -a30
```
as well, this would require small changes to the test.

Reply:
added